libr/util/buf.c

```diff
@@ -287,6 +287,22 @@ R_API int r_buf_set_bytes(RBuffer *b, const ut8 *buf, ut64 length) {
 	if (!(b->buf = malloc (length)))
 		return false;
 	memmove (b->buf, buf, length);
+
+	b->length = length;
+	b->empty = 0;
+	return true;
+}
+
+R_API int r_buf_set_bytes_steal(RBuffer *b, const ut8 *buf, ut64 length, bool steal_ptr) {
```

Comment:
shouldnt this function just steal the pointer? whats the purpose of the bool parameter if there's another function that already does the set_byets() without stealing pointers?